src/Features/Core/Portable/IncrementalCaches/SymbolTreeInfoIncrementalAnalyzerProvider.cs

```diff
@@ -319,7 +320,8 @@ private bool SupportAnalysis(Project project)
                     return false;
                 }
 
-                return RemoteFeatureOptions.ShouldComputeIndex(project.Solution.Workspace);
+                return RemoteFeatureOptions.ShouldComputeIndex(project.Solution.Workspace) &&
+                    !ServiceFeatureOnOffOptions.IsPowerSaveModeEnabled(project);
```

Comment:
i don't think we want this.  Instead what i think we shoudl do is *not* precalculate things when we're in powersaving mode.  this looks like this would break all symbol-tree-info features, even if the user explicitly asks to use it.

instead, i think it makes more sense to make sure those features don't cause *undue* power usage unless explicitly requested by the user.

Reply:
Ah, so basically just guard the incremental analyzer in this file for this option, but not the entire SupportsAnalysis functionality. That seems reasonable.

Reply:
Exactly! :D